add some template emails (#22)
cct-datascience/repro-data-sci: emails.md

## Accepted

> Subject: "Reproducibility and Data Science in R Workshop Series"
>
> Congratulations!
> You have been accepted into the **Reproducibility and Data Science in R** [Fall workshop series](https://datascience.cct.arizona.edu/events/203-fall-2023-workshop-series-reproducibility-and-data-science-r).
> The short course will meet every Tuesday and Thursday from 11am to 1pm from \<start\> through \<end\> ending with a reproducibility colloquium on \<date\> to allow you to showcase what you’ve learned.
> **Please let us know if you can no longer commit to attending most sessions** by \<final deadline\> so we can open up spaces for additional applicants.
>
> Keep an eye out for a follow up email in the next couple of weeks with more
> details including an invite to our Slack channel.
> We are looking forward
> to meeting you all, and in the meantime, please don’t hesitate to reach
> out with any questions you might have.

## Not accepted

> Subject: "Reproducibility and Data Science in R Workshop Series"
>
> Dear applicant,
>
> Thank you for applying to our Fall Reproducibility and Data Science in R workshop series.

> > After careful review of all applicants, unfortunately your application has not been selected for participation this year.

Reviewer: ```suggestion
> We received a large number of applications this year, and, unfortunately, your application was not selected for participation in this year's workshop. 
```

> > We received many applications and unfortunately we were not able to accept everyone—a difficult decision for us.

Reviewer: ```suggestion
> 
```

> We encourage you to apply again next year (and remind us that you applied previously when you do) and to continue to reach out to us with data science and reproducibility questions via email, our weekly [drop-in hours](https://datascience.cct.arizona.edu/drop-in-hours), and in the UA Data Science [Slack group](https://jcoliver.github.io/uadatascience-slack/).
>
> If you’re up for doing some self-study in lieu of attending the workshop series this Fall, here are some suggested places to get started:

> >

Reviewer 2: ```suggestion
> 
> - Fall workshop series content via the [website for the series](https://cct-datascience.github.io/repro-data-sci/)
> 
```

Author: @diazrenata how do you feel about pointing applicants that aren't accepted toward the workshop materials?
